src/api/routes/tx.ts

```diff
@@ -75,7 +75,7 @@ export function createTxRouter(db: DataStore): RouterWithAsync {
 
     // TODO: use getBlockWithMetadata or similar to avoid transaction integrity issues from lazy resolving block tx data (primarily the contract-call ABI data)
     const results = await Bluebird.mapSeries(txResults, async tx => {
-      const txQuery = await getTxFromDataStore(db, { txId: tx.tx_id, includeUnanchored });
+      const txQuery = await getTxFromDataStore(db, { txId: tx.tx_id, dbTx: tx, includeUnanchored });
```

Comment:
Shouldn't this be `includeUnanchored: includeUnanchored` ?

Reply:
As opposed to the shorthand of `includeUnanchored`? It's only a stylistic difference. I don't feel strongly either way

Reply:
Oh, I think I just hadn't seen that shorthand before 👍 all good